src/app/component/Slider/Slider.component.js

```diff
@@ -503,7 +503,7 @@ class Slider extends Component {
         );
     }
 
-    renderSlide(block, item, id, listItemWidth, arePlaceholdersShown) {
+    renderSlide(block, item, id, listItemWidth, arePlaceholdersShown, showGreyPlaceholder) {
         return (
```

Comment:
Let's destruct item, because we are not really using the `item` itself. Plus there are a lot of references to `item` properties.

Reply:
item is used in line 512 for function getKey().

Reply:
CR comment fixed.